README.md

```diff
@@ -1,7 +1,204 @@
 enb-css
```

Comment:
I think some notes about source maps in this file will be good.
But I'd move info about `ie.css` into separate file.